docs/dashboard.asciidoc

```diff
@@ -3,66 +3,86 @@
 
 [partintro]
 --
-A Kibana _dashboard_ displays a collection of visualizations, searches, and {kibana-ref}/maps.html[maps].
-You can arrange, resize, and edit the dashboard content and then save the dashboard
-so you can share it.
 
-[role="screenshot"]
-image:images/Dashboard_example.png[Example dashboard]
+A {kib} _dashboard_ is a collection of visualizations, searches, and 
+maps, typically in real-time. Dashboards provide 
+at-a-glance insights into your data and enable you to drill down into details. 
 
---
+To start working with dashboards, click *Dashboard* in the side navigation. 
+You can:
 
-[[dashboard-getting-started]]
-== Building a Dashboard
+* <<dashboard-create-new-dashboard, Create a dashboard>>
+* <<customizing-your-dashboard, Arrange dashboard elements>>
+* <<sharing-dashboards, Share a dashboard>>
+* <<import-dashboards, Import and export dashboards>>
+* <<viewing-detailed-information, Inspect and edit dashboard elements>>
+
+
+[role="screenshot"]
+image:images/Dashboard_example.png[Example dashboard]
 
-If you haven't yet indexed data into {es} or created an index pattern,
-you'll be prompted to do so as you follow the steps for creating a dashboard.
-Or, you can use one of the prebuilt sample data sets, available from the
-Kibana home page.
 
 [float]
 [[dashboard-read-only-access]]
 === [xpack]#Read only access#
-When you have insufficient privileges to create or save dashboards, the following 
-indicator in Kibana will be displayed. The buttons to create new dashboards or edit
-existing dashboard won't be visible. For more information on granting access to
-Kibana see <<xpack-security-authorization>>.
+If you see
+the read-only icon in the application header,
+then you don't have sufficient privileges to create and save dashboards. The buttons to create and edit
+dashboards are not visible. For more information, see <<xpack-security-authorization>>.
 
 [role="screenshot"]
 image::images/dashboard-read-only-badge.png[Example of Dashboard's read only access indicator in Kibana's header]
 
 [float]
+[[dashboard-getting-started]]
+=== Interact with dashboards
+
+When you open *Dashhboard*, you're presented an overview of your dashboards. 
+If you don't have any dashboards, you can add 
+<<add-sample-data, sample data sets>>,
+and check out the pre-built dashboards. 
+
+Once you open a dashboard, you can filter the data
+by entering a search query, changing the time filter, or clicking 
+in the visualization, search, or map. If a dashboard element has a stored query, 
```

Comment:
wouldn't that be "clicking in **a** visualization, search, or map" or "clicking in the visualization**s**, search**es**, or map**s**"?